templates/common/_base/units/machine-config-daemon-firstboot.service.yaml

```diff
@@ -14,6 +14,7 @@ contents: |
   [Service]
   Type=oneshot
   RemainAfterExit=yes
+  ExecStart=/usr/bin/sh -c "sed -i 's/enabled=1/enabled=0/' /etc/yum.repos.d/*.repo"
```

Comment:
comment will be nice for future reference

Reply:
Good idea, fixed